vng_api_common/geo.py

```diff
@@ -43,6 +43,9 @@ def perform_crs_negotation(self, request):
             if content_crs != DEFAULT_CRS:
                 raise NotAcceptable(detail=f"CRS '{content_crs}' is niet ondersteund")
 
+        if self.action == "destroy":
```

Comment:
a bit more robust would be:

```suggestion
        if request.method.lower() == "delete":
```

This would then also work with regular `APIView`